assets/css/twenty-twenty-one.scss

```diff
@@ -2432,6 +2473,23 @@ a.reset_variations {
 		}
 	}
 
+	/**
+     * Home page
+     */
+	.home #main {
+
+		.wp-block-cover {
+			max-width: none;
```

Comment:
I'm wondering if we should modify styles outside of a `woocommerce` class... in theory, this would potentially affect the cover for non-WooCommerce homepages.

I also removed this and didn't notice any difference.